dbms/src/Common/ProfileEvents.cpp

```diff
@@ -90,6 +90,8 @@
     M(CompileSuccess) \
     \
     M(CompileFunction) \
+    M(CompiledFunctionExecute) \
+    M(CompiledCacheSizeBytes) \
```

Comment:
We can place it in AsynchronousMetrics, like it is done for MarkCache, UncompressedCache.
Better to not reset ProfileEvents manually because they are supposed to be incremental counters.